src/test/java/com/google/jenkins/plugins/storage/StringJsonServiceAccountConfig.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jenkins.plugins.storage;
+
+import com.google.api.client.json.jackson.JacksonFactory;
+import com.google.api.client.util.PemReader;
+import com.google.common.base.Strings;
+import com.google.jenkins.plugins.credentials.oauth.JsonKey;
+import com.google.jenkins.plugins.credentials.oauth.KeyUtils;
+import com.google.jenkins.plugins.credentials.oauth.ServiceAccountConfig;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/** Provides a means for read/writing JSON service account config files for integration tests. */
+// TODO: (rachelyen): remove once OAuth plugin upgrades >= 0.8
```

Comment:
Please be sure to link to an issue for the repo to track the work.

Reply:
Done